aclk/aclk_lws_https_client.c

```diff
@@ -38,6 +38,7 @@ static int simple_https_client_callback(struct lws *wsi, enum lws_callback_reaso
         n = perconn_data->data_size;
         if (lws_http_client_read(wsi, &ptr, &n) < 0)
             return -1;
+        ptr[n] = '\0';
```

Comment:
This writes off the end of the allocated space, change L38 to  `n = perconn_data->data_size-1`;

Reply:
The n is overwritten by `lws_http_client_read` to the actual bytes read. That's what I want null terminated.

Reply:
Yes, I understand that. But the contract with `lws_http_client_read` is that it can read up to `n` bytes, so if it fills the buffer provided then `n` will be unchanged and the null will be written off the end of the allocated space.

Reply:
You are right I will give it one less byte

Reply:
Pushed the change